src/lib/components/Footer/FooterIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, PropsWithChildren, ComponentProps } from 'react';
+import classNames from 'classnames';
+
+export type FooterIconProps = PropsWithChildren<{
+  href?: string;
+  className?: string;
+  icon: FC<ComponentProps<'svg'>>;
+  ariaLabel?: string;
+}>;
+
+export const FooterIcon: FC<FooterIconProps> = ({ href, ariaLabel, className, icon: Icon }) => {
+  return (
+    <>
+      {href ? (
+        <a href={href} aria-label={ariaLabel} className={classNames('text-gray-500  dark:hover:text-white', className)}>
+          <Icon className="h-5 w-5" />
+        </a>
+      ) : (
+        <Icon className="h-5 w-5" aria-label={ariaLabel} />
```

Comment:
When there is just an icon - no link - arguably the icon is serving a purely presentation purpose, and should just be hidden to screen readers

Reply:
@tulup-conner I left this icon option without a link because I believe have the possibility to only add only icons. Like this example. What do you think?
![image](https://user-images.githubusercontent.com/41678283/166925298-6ffe4778-b0bf-48f9-92ef-6810a1ed76f5.png)

Reply:
Ah got it, I'll fix it.